### ISSUE TITLE
deploy instructions

### ISSUE THREAD

**Issue by hotsjf** [popcorn-time-ru/popcorn-ru#5]
Hey 
 
Is it possible that you write up an instruction for how to deploy this API

Like how to setup the scrappers and DB stuff, and how to set up the rewrite rules etc.

thanks

**Comment by ivan1986** [comment]
Added to readme

**Comment by hotsjf** [comment]
Thanks for the instructions 

have all then environments set up

now when I try to run
`pgrep -c -f enqueue:consume || bin/console enqueue:consume --time-limit="now + 55 minutes" --no-debug --memory-limit=200`

got quite a lot of errors

I think the first question would be, how to set up the tor proxy ? I'm running on CentOS7

**Comment by ivan1986** [comment]
mmm
```
yum install epel-release
yum install tor
```
and what errors you get? for now only rutor use tor proxy
try `bin/console enqueue:consume -vvv --logger=stdout` for debug

**Comment by hotsjf** [comment]
> mmm
> 
> ```
> yum install epel-release
> yum install tor
> ```
> 
> and what errors you get? for now only rutor use tor proxy
> try `bin/console enqueue:consume -vvv --logger=stdout` for debug

I believe it's the Tor connection errors. I did installed Tor, but I'm not really sure how to config it, I suppose the spider need Tor proxy to crawl some data?

here is one of the errors

`[debug] [DoctrineClearIdentityMapExtension] Clear identity map for manager "default"
cURL error 7: Failed connect to 127.0.0.1:9050; Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)`

**Comment by ivan1986** [comment]
127.0.0.1:9050 is default tor config as i know, `systemctl status tor` and start, and if not help check port and change in .env.local

**Comment by hotsjf** [comment]
Seems Tor needs 0 config lol, and I did too much 

now seems the initial fill is running, I got lots fo these errors, is that normal? or I have something config incorrectly ?

`05:17:03 WARNING   [app] Not found media
[
  "imdb" => "tt12455646",
  "extra" => [
    "imdb" => "tt12455646"
  ]
]`

**Comment by ivan1986** [comment]
film exist
do you configure api keys for tmdb and also for track?
`TMDB_API_KEY=`

**Comment by ivan1986** [comment]
ha, ups, in tmdb film not exist

**Comment by hotsjf** [comment]
yeah, I did config the tmdb_api, so that's supposed to be like that?

I'll get to know this project in more details, after that will see what I can contribute 

thanks for the good work

**Comment by ivan1986** [comment]
yes, it's ok, it's new not popular film - need update tmdb

thanks you for help, for test deploy, and for interest to project

**Comment by hotsjf** [comment]
and one more question, one of the command in the cron:
`bin/console sync:old 180 500 --days-check=180 --days-delete=360`

I got this error:

`  Too many arguments to "sync:old" command, expected arguments "limit".`

**Comment by ivan1986** [comment]
thanks, fixed, it's was `days-check` before added `days-delete`

`bin/console sync:old 500 --days-check=180 --days-delete=360`
